src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -808,6 +811,33 @@ private void paste() {
         }
     }
 
+    private void copyTitle() {
+        List<BibEntry> bes = mainTable.getSelectedEntries();
+        if (!bes.isEmpty()) {
+            storeCurrentEdit();
+            List<String> titles = new ArrayList<>(bes.size());
+
+            // Collect all non-null keys.
+            for (BibEntry be : bes) {
+                be.getTitle().ifPresent(titles::add);
```

Comment:
Leave it like this, my comment was just nitpicking. I was referring to our [toStream method](https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/model/util/OptionalUtil.java#L24).